app/frontend/controllers/app_controller.js

```diff
@@ -480,20 +524,22 @@ class AppController {
       return false;
     });
   }
-
-  toggleFullScreen() {
+  
+  getPlatform() {
     var platform = null;
 
     if (platformHelper.isElectron()) {
-
       platform = electronPlatform;
-
     } else if (platformHelper.isAndroid()) {
-
       platform = cordovaPlatform;
-
     }
 
+    return platform;
+  }
```

Comment:
I think it's a good method to move to `ezraHelper` module.

Ideally, `app_controller` can be the main container without much logic.

Reply:
Ok. Will move it to platform helper directly.